src/visor/blockchain.go

```diff
@@ -235,20 +263,25 @@ func (bc *Blockchain) ExecuteBlock(b *coin.Block) (coin.UxArray, error) {
 	txns := b.Body.Transactions
 	for _, tx := range txns {
 		// Remove spent outputs
-		bc.unspent.DelMultiple(tx.In)
+		if err := bc.Unspent.Delete(tx.In); err != nil {
```

Comment:
does it need to rollback previous changes if it encounters error partway through?